### PR TITLE
update dotnet documentation path

### PR DESCRIPTION
**Pull request by vikpande**
Apparently, this page is also moved and now not maintained "https://hyperledger.github.io/indy-sdk/wrappers/dotnet/docs/index.html". However, I located the documentation link in within the repository and hence updated it here. Please verify and approve/replace with the new link. Thank you. 

Signed-off-by: Vikas Pandey <vikpande@gmail.com>

### CODE REVIEW

File: wrappers/dotnet/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Pull requests welcome!
 
 ### Documentation
 
-Documentation for the .NET wrapper is available [here](http://hyperledger.github.io/indy-sdk/wrappers/dotnet/docs/index.html).
+Documentation for the .NET wrapper is available [here](https://github.com/hyperledger/indy-sdk/blob/master/wrappers/dotnet/docs/index.html).
```
Comment (jovfer) [inline wrappers/dotnet/README.md]: as far as I remember the initial plan was to use github.io to render .NET documentation. and there were some issues with it. Like `.io` is not enabled for HL/indy-sdk repo. And may be something else.
@srottem @tmarkovski could you share more details about this topic

Comment (vikpande) [reply wrappers/dotnet/README.md]: sure @jovfer . waiting for their inputs. thanks.

Comment (srottem) [reply wrappers/dotnet/README.md]: Correct.  The repo needs to be configured with "Github Pages" with the source as the Master branch. See here for details: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/.

Comment (vikpande) [reply wrappers/dotnet/README.md]: thanks @srottem , guess maintainers/admin can do that, since i dont have access to it :-)

Comment (srottem) [reply wrappers/dotnet/README.md]: Me neither.  We've been waiting for this to happen for more than a year now and it's been brought up on RocketChat multiple times, so I'm guessing no one wants to do it.

Comment (jovfer) [reply wrappers/dotnet/README.md]: As far as I know there are 2 main blockers:
- no one in core team has permission to enable .io
- probably there are some contradictions between this documentation and HIPE https://github.com/hyperledger/indy-hipe/tree/master/text/0025-indy-docs-framework

The first one seems just technical question for HL-desk, as I have seen some .io for others HL projects.
I will figure out how to resolve the second one.
@srottem Are you available in rocket.chat of sovrin or hyperledger for this discussion?

Comment (vikpande) [reply wrappers/dotnet/README.md]: thanks guys @srottem @jovfer . I am in the sovrin working group and will flag this in the call today. will update you today about the inputs i receive.

**Comment by jovfer** [comment]
Hi @michaeldboyd 
You have worked on ReadTheDocs adoption for IndySDK documentation. Could you provide some feedback please:
1) Are we going to store developer API documentation together with other overviews?
2) If so, is it possible to incapsulate .net documentation into current readthedocs page about IndySDK?

**Comment by kdenhartog** [comment]
@jovfer I just got word back from @tmarkovski 

```
It's safe to merge this PR, it's really just a path upgrade. The online documentation won't be the go-to source for docs, as there is XML doc already generated and available with the nuget package.

PR does not impact runtime configuration at all
```

**Pull request merged**